architecture/ECOSYSTEM-VIEW.md

```diff
@@ -0,0 +1,15 @@
+#Openline Ecosystem 
+
+The openline ecosystem is composed out of 3 main parts : Customer OS, Components and Applications.
+
+## Customer OS
+A collection of Services that offer a simple standard of interacting with Customer data. These services are built on top of multiple types of datasources and offer a complete view of the customer journey. 
+
+## Components
+In order to make it easier to interact with the Customer OS, we are proposing a set of most needed components in the interactions with your customers: Forms, Authentication, Conversation Views etc. 
+
+## Applications
+Using the components and our Customer OS you can build applications faster and then you can then register them in Customer OS to have access to all your customer data. 
+
+
+![alt text](https://raw.githubusercontent.com/openline-ai/openline-customer-os/27-document-ecosytem-view-for-customer-os/architecture/images/customer-os-eco-system-view.png)
```

Comment:
Communication App instead of Conversation App?

i think we can add another square for: expose changes to the world ( push events to outside when data is changed )